.all-contributorsrc

```diff
@@ -922,6 +922,15 @@
       "contributions": [
         "translation"
       ]
+    },
+    {
+      "login": "olets",
+      "name": "Henry Bley-Vroman",
+      "avatar_url": "https://avatars.githubusercontent.com/u/3282350?v=4",
+      "profile": "http://olets.dev",
```

Comment:
Can we change this to https, here and in the README? Lmk if I can help.

Wonder if that's a GitHub thing or an all-contributors thing. In my profile I had no protocol 

<kbd>![image](https://user-images.githubusercontent.com/3282350/210886776-471a9f2b-2365-49ff-b2eb-44947bddfe3d.png)</kbd>

I've updated my profile to add https:// so closing this PR and making a new one should be one way of making the change.

Reply:
That's pulled from GH's API. If that's the only change you made, then I can simply rectify it on this PR.

Reply:
Good to know!

Yep that's the only change